src/note-components/notepage/note.css

```diff
@@ -1,109 +1,162 @@
-body{
+
+body {
     font-family: 'Montserrat', sans-serif;
+    margin-bottom: 3.125rem;
 }
-.main-container{
+
+.main-container {
     display: flex;
 }
-.left-nav{
+
+/* Sidebar CSS */
+.left-nav {
+    text-align: center;
     width: 20%;
     align-self: auto;
     box-shadow: 2px 2px 6px 0px rgba(0, 0, 0, 0.1);
 }
-.right-cont{
-    width: 80%;
-    margin-top: 30px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    flex-direction: column;
-    gap: 20px;
-}
-.left-list{
+
+.left-list {
     list-style: none;
     font-size: larger;
     padding-left: 5px;
 }
-.left-item{
-    margin: 15px;
+
+.left-item {
+    margin: 0.9375rem;
     padding: 10px;
     border-radius: 5px;
     color: var(--primary-color);
 }
-.left-item:hover{
+
+.left-item:hover {
     background-color: #b6bfc9;
 }
-.fal{
+
+/* Right Flex Container Main */
+
+.right-cont {
+    width: 80%;
+    margin-top: 1.875rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    gap: 1.25rem;
+}
+
+.fal {
     color: var(--primary-color);
 }
-span{
+
+span {
     margin-right: 10px;
 }
-/* .note-container{
-    display: flex;
-} */
-.note-container{
+
+/* New Note Container Properties */
+
+.note-container {
     background-color: #b6bfc9;
     border-radius: 10px;
-    padding: 10px ;
-    width: 500px;
-}
-.note-text{
-    color: var(--primary-color);
     padding: 10px;
-    background-color: transparent;
-    border: none;
-    height: 80px;
-    width: 500px;
-    font-size: large;
+    width: 31.25rem;
+}
+
+.note-head {
+    display: flex;
+    gap: 1.25rem;
 }
-.title-note{
-    width: 400px;
+
+.note-text {
+    width: 25rem;
     color: var(--primary-color);
     padding: 10px;
     background-color: transparent;
     border: none;
-    height: 80px;
+    height: 5rem;
     font-size: large;
+    text-transform: capitalize;
 }
-.note-text:focus, .title-note:focus{
-    outline: none;
-}
-.pin{
-    margin-left: auto;
-}
-.note-head{
-    display: flex;
-    gap: 20px;
-}
-.note-foot{
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    gap: 20px;
-}
-.foot-icons{
-    display: flex;
-    gap: 10px;
-    margin-left: auto;
-}
+
 .date-inp,
-.note-title{
+.note-title {
     margin-left: 10px;
     padding: 5px;
     border-radius: 5px;
     background-color: transparent;
 }
-.date-inp{
+
+.date-inp {
     border: 1px solid var(--primary-color);
 }
-.note-title{
+
+.note-title {
     border: none;
     font-size: larger;
     font-family: 'Montserrat', sans-serif;
     font-weight: bold;
     color: var(--primary-color);
+    text-transform: capitalize;
+}
+
+.note-title:focus {
+    outline: none;
 }
-.note-title:focus{
+
+.note-text:focus {
     outline: none;
-}
\ No newline at end of file
+}
+
+/* Created Note  */
+
+.note-list {
+    border-radius: 10px;
+    padding: 10px;
+    background-color: #133863;
```

Comment:
use variables

Reply:
This color was used just once, that is why I didn't make a variable out of it.